projects/boggle/public/scripts/script.js

```diff
@@ -1,27 +1,59 @@
-let letters = [AAEEGN, ABBJOO, ACHOPS, AFFKPS, AOOTTW, CIMOTU, DEILRX, DELRVY, DISTTY, EEGHNW, EEINSU, EHRTVW, EIOSST, ELRTTY, HIMNUQu, HLNNRZ]
+let letters = ['AAEEGN', 'ABBJOO', 'ACHOPS', 'AFFKPS', 'AOOTTW', 'CIMOTU', 'DEILRX', 'DELRVY', 'DISTTY', 'EEGHNW', 'EEINSU', 'EHRTVW', 'EIOSST', 'ELRTTY', 'HIMNUQu', 'HLNNRZ'];
 
-const stringletters = (s, array) => {
+const stringletters = (s) => {
   let array = [];
   for (let i = 0; i < s.length; i++) {
+    if (s[i] === 'Q'){
+      array.push('Qu')
+      i++
+    }
     array.push(s[i]);
-  }
+  };
   return array;
-}
+};
 
 const diceArrays = () => {
   let dice = [];
   for (let i = 0; i < letters.length; i++) {
     dice.push(stringletters(letters[i]));
-  }
+  };
   return dice;
-}
+};
 
 const boardArray = () => {
-  let board = [['','','',''],['','','',''],['','','',''],['','','','']];
+  let board = [['', '', '', ''], ['', '', '', ''], ['', '', '', ''], ['', '', '', '']];
   for (let i = 0; i < 4; i++) {
     for (let j = 0; j < board.length; j++) {
-      let board[i, j] = letters[j, math.floor(math.random * 5)]
-    }
+      board[i][j] = letters[j][Math.floor(Math.random * 5)];
```

Comment:
You're missing the `()`s on the call to `Math.random`

```suggestion
      board[i][j] = letters[j][Math.floor(Math.random() * 5)];
```

Consequently you're multiplying 5 times a function which is probably producing `NaN` (a.k.a. not a number) which then is not a valid index into the string at `letters[j]` thus you're setting `board[i][j]` to undefined. This is just a detail that you have to get really used to: when you're calling a function or method you always have to have `()`s. If you don't you're accessing a variable which is legal but if the value of the variable is a function it's probably not what you want.